jcl/source/windows/JclDebug.pas

```diff
@@ -2331,6 +2346,25 @@ function TJclMapScanner.SourceNameFromAddr(Addr: DWORD): string;
   end;
 end;
 
+function TJclMapScanner.VAFromUnitAndProcName( UnitName, ProcName : string ): DWORD;
+var
+  I : Integer;
+begin
+  Result := 0;
+  if ( UnitName = '' ) or ( ProcName = '' ) then
+    Exit;
+  ProcName := UnitName + '.' + ProcName;
+
+  for i := Low( FProcNames ) to High( FProcNames ) do
+    begin
+    if ( CompareText( MapStringCacheToStr( FProcNames[ I ].ProcName, True ), {UnitName + '.' + }ProcName ) = 0 ) then

```

Comment:
No space between parenthesis
No space between brackets
There should not be any commented out code here